recipes/happi/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "happi" %}
+{% set version = "1.4.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/pcdshub/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: 6135928358a329e715758624cfcc95c902d4ddc936bc8e29dd2d356874611153
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+  run:
+    - python >=3.6
+    - jinja2
+    - prettytable
+    - simplejson
+    - coloredlogs
+
+test:
+  imports:
+    - happi 
+
+about:
+  home: https://github.com/pcdshub/happi 
+  license: SLAC Open
+  license_family: OTHER
+  license_file: LICENSE.md
+  summary: 'Heuristic Access to Positions of Photon Instruments'
+
+extra:
+  recipe-maintainers:
+    - hhslepicka
+    - klauer
+    - ZLLentz
```

Comment:
@ZLLentz are you OK with being a maintainer?

Reply:
I agree to be a maintainer